src/Makefile.qt.include

```diff
@@ -360,6 +361,7 @@ QML_RES_QML = \
   qml/components/StorageSettings.qml \
   qml/components/ThemeSettings.qml \
   qml/components/TotalBytesIndicator.qml \
+  qml/components/Tooltip.qml \
```

Comment:
forgot to add this?

Reply:
whoops

Reply:
thank you, updated the pr with the necessary files